polaris-react/src/components/BulkActions/components/BulkActionButton/BulkActionButton.tsx

```diff
@@ -36,29 +37,38 @@ export function BulkActionButton({
     }
   });
 
-  const buttonContent =
-    disclosure && !showContentInButton ? undefined : content;
+  const shouldShowDotsIcon = disclosure && !showContentInButton;
```

Comment:
Could this variable be renamed? `shouldShowDotsIcon`? It doesn't tell much about what it is, perhaps something more along the lines of what is represents. I am not sure in this case, but does it represent `showMoreActionsIcon`?